### PR TITLE
feat: Modified UI for input (#1715)

### LINKED ISSUE #1715: Improve UI of VoiceInput Fragment
**Actual Behaviour**

Looks quite bland and feels detached.
**Screenshots of the issue**

voice fragment]
**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by kartik-karz**
Fixes #1715 
Changes:
* Input mode has been made to look like a single button.
* Voice fragment has been made transparent and few code modifications have been done to accommodate a keyboard button for the original voice input.

Screenshots for the change: 
Normal
normal

with input
with_input

with keyboard
with_keyboard

PS: I know that it's required that we make single PR for one feature but both of them were interlinked and modifiying one would require someother temp changes and to prevent that this is done.

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 1]
```diff
@@ -98,6 +100,17 @@ class ChatActivity : AppCompatActivity(), IChatView {
                 chatPresenter.startComputingThread()
             }
         }
+        
+        val cardViewWithK = findViewById<CardView>(R.id.with_keyboard)
+        val cardViewWithoutK = findViewById<CardView>(R.id.without_keyboard)
+        val keyboard = findViewById<View>(R.id.btn_keyboard)
+        keyboard.setOnClickListener {
+            if(!input_held){
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Spaces around the parentheses missing. Please reformat the code for the changes made.

**Comment by kartik-karz** [comment]
Sorry was busy last week, will try to resolve it this week

**Comment by atm1504** [comment]
@iamareebjamal @batbrain7 @arundhati24  I think this issue should be closed as we have already got many changes in the ui.

**Comment by kartik-karz** [comment]
Cool @iamareebjamal doing it now

**Pull request closed without merging**